packages/runtime/src/runtimes/react/pollBoxModel.tsx

```diff
@@ -0,0 +1,32 @@
+import { BoxModel, getBox } from '../../state/modules/box-models'
```

Comment:
open to suggestions as to where this should go 🤷🏼 

We don't really have a folder for utils like this that aren't hooks.

Reply:
We might already have some code that does something like this so we could re-use it. If it makes sense for it to be in a shared file then we can figure out some structure for it

Reply:
Ok, it seems to have been extracted from the Slot control. I think it's good where it is right now, but can we follow the file-naming conventions from other utilities in that folder and rename to `poll-box-model.ts`? (note the lack of `x` in the extension 😄 )